README

```diff
@@ -1,7 +1,17 @@
 Evaluation
 ==========
 
- Evaluation module is one of essential module for a search engine to be used in academia.This module ease to calculate various metric  precision, recall, precision @ rank,Precision @ recall which validates implementation and check the engine on result presented on literature.xapian being an open source engine have implemented evaluation module to help people from academia.
+Evaluation module is one of essential module for a search engine to be used in academia. This module eases to calculate various metric precision, recall, precision @ rank, Precision @ recall which validates implementation and check the engine on result presented on literature. Xapian being an open source engine have implemented evaluation module to help people from academia.
+
+Buiding Evaluation Module
+-------------------------
+
+User needs to build Evaluation module on their system after cloning the source repository. This can be done using
+::
+ autoreconfig -fiv
```

Comment:
The command is `autoreconf` not `autoreconfig`.

Reply:
Sorry, I have updated it now.